File: PRAK110-2210817210023-Muhammad Shofy Akmal.py
import math
a,c=12,5
print ("Diketahui : ")
print ("Alas = ", c,"cm")
print ("Tinggi = ", a,"cm")
b=round(math.sqrt(a*a+c*c))
print ("Jawab : ")
print ("Sisi A = ", a,"cm")
print ("Sisi B = ", b,"cm")
print ("Sisi C = ", c,"cm")
print ("Keliling = ",a+b+c,"cm") 
print ("Luas = ",round((a*c)/2),"cm")

File: PRAK110-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
#include <math.h>
int main () {
int a=12, c=5, b;
printf ("Diketahui : \n");
printf ("Alas = %d cm\n", c);
printf ("Tinggi = %d cm\n", a);
b=sqrt(a*a+c*c);
printf ("Jawab : \n");
printf ("Sisi A = %d cm\n", a);
printf ("Sisi B = %d cm\n", b);
printf ("Sisi C = %d cm\n", c);
printf ("Keliling = %d cm\n",a+b+c); 
printf ("Luas = %d cm", (a*c)/2);
}